tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -4,36 +4,34 @@
 using System;
 using System.IO;
 using System.Linq;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Bot.Builder.Adapters;
-using Microsoft.Bot.Builder.Tests;
 using Microsoft.Bot.Schema;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using RichardSzalay.MockHttp;
 
 namespace Microsoft.Bot.Builder.Ai.Luis.Tests
 {
     [TestClass]
-    [Ignore]
     // The LUIS application used in these unit tests is in TestData/TestLuistApp
     public class LuisRecognizerTests
     {
-        private readonly string appId = TestUtilities.GetKey("LUISAPPID");
-        private readonly string subscriptionKey = TestUtilities.GetKey("LUISAPPKEY");
-        private readonly string region = "Westus";
+        private const string _luisAppId = "ab48996d-abe2-4785-8eff-f18d15fc3560";
+        private const string _subscriptionKey = "cc7bbcc0-3715-44f0-b7c9-d8fee333dce1";
+        private const string _region = "westus";
```

Comment:
Did you check for style cop warnings?  I don't think we allow _ names anymore.  This is a test class so may not matter, but still.